### DISCUSSION TITLE
How to update Meshcentral?

### DISCUSSION THREAD

**Discussion by NorthernMunkey** [tteck/Proxmox#2587] [Help Needed]
it's on 1.1.5 - but there's several versions to 1.2.1, available,  calling the update via the webgui fails.  it may have been asked but what's the way to update this if there is any? and if not will recreation pull the latest?

**Comment by tteck** [comment]
Try executing the command `npm install -U meshcentral` in the LXC console, afterwards reboot. Please share your findings.

**Comment by jimcuk** [comment]
I tried this and get npm: command not founf
